cli/printer/text_test.go

```diff
@@ -219,72 +230,77 @@ func ExampleTextPrintServices() {
-			ServiceName:      "svc1",
+			Deployments: []models.Deployment{
+				{DeployName: "d1", DeployVersion: "1"},
+			},
+			DesiredCount:     1,
 			EnvironmentID:    "eid1",
 			EnvironmentName:  "ename1",
 			LoadBalancerID:   "lid1",
 			LoadBalancerName: "lname1",
 			RunningCount:     1,
-			DesiredCount:     1,
-			Deployments: []models.Deployment{
-				{DeployName: "d1", DeployVersion: "1"},
-			},
+			ServiceID:        "id1",
+			ServiceName:      "svc1",
+			ServiceType:      models.DeployCompatibilityStateless,
 		},
 		{
-			ServiceID:      "id2",
-			ServiceName:    "svc2",
-			EnvironmentID:  "eid2",
-			LoadBalancerID: "lid2",
-			RunningCount:   1,
-			DesiredCount:   1,
 			Deployments: []models.Deployment{
 				{DeployID: "d2.1"},
 			},
+			DesiredCount:   1,
+			EnvironmentID:  "eid2",
+			LoadBalancerID: "lid2",
+			RunningCount:   1,
+			ServiceID:      "id2",
+			ServiceName:    "svc2",
+			ServiceType:    models.DeployCompatibilityStateful,
 		},
 		{
-			ServiceID:     "id3",
-			ServiceName:   "svc3",
-			EnvironmentID: "eid3",
-			RunningCount:  0,
-			DesiredCount:  1,
-			PendingCount:  1,
 			Deployments: []models.Deployment{
 				{DeployID: "d3.1", RunningCount: 0, DesiredCount: 1},
 			},
+			DesiredCount:  1,
+			EnvironmentID: "eid3",
+			PendingCount:  1,
+			RunningCount:  0,
+			ServiceID:     "id3",
+			ServiceName:   "svc3",
+			ServiceType:   models.DeployCompatibilityStateless,
 		},
 		{
-			ServiceID:     "id4",
-			ServiceName:   "svc4",
-			EnvironmentID: "eid4",
-			RunningCount:  1,
-			DesiredCount:  2,
-			PendingCount:  1,
 			Deployments: []models.Deployment{
 				{DeployID: "d4.1", RunningCount: 1, DesiredCount: 2},
 			},
+			DesiredCount:  2,
+			EnvironmentID: "eid4",
+			PendingCount:  1,
+			RunningCount:  1,
+			ServiceID:     "id4",
+			ServiceName:   "svc4",
+			ServiceType:   models.DeployCompatibilityStateful,
 		},
 		{
-			ServiceID:     "id5",
-			ServiceName:   "svc5",
-			EnvironmentID: "eid5",
-			RunningCount:  2,
-			DesiredCount:  1,
 			Deployments: []models.Deployment{
 				{DeployID: "d5.1", RunningCount: 1, DesiredCount: 0},
 				{DeployID: "d5.2", RunningCount: 0, DesiredCount: 1},
 			},
+			DesiredCount:  1,
+			EnvironmentID: "eid5",
+			RunningCount:  2,
+			ServiceID:     "id5",
+			ServiceName:   "svc5",
+			ServiceType:   models.DeployCompatibilityStateful,
 		},
 	}
 
 	printer.PrintServices(services...)
 	// Output:
-	// SERVICE ID  SERVICE NAME  ENVIRONMENT  LOADBALANCER  DEPLOYMENTS  SCALE
-	// id1         svc1          ename1       lname1        d1:1         1/1
-	// id2         svc2          eid2         lid2          d2:1         1/1
-	// id3         svc3          eid3                       d3:1*        0/1 (1)
-	// id4         svc4          eid4                       d4:1*        1/2 (1)
-	// id5         svc5          eid5                       d5:1*        2/1
+	// SERVICE ID  SERVICE NAME  ENVIRONMENT  LOADBALANCER  DEPLOYMENTS  SCALE    TYPE
```

Comment:
[nit] this is a little hard to differentiate at a glance, perhaps the column is `STATEFUL` and the rows are `true/false`?

Reply:
✅